modules/analysis-common/src/test/java/org/elasticsearch/analysis/common/CommonGramsTokenFilterFactoryTests.java

```diff
@@ -251,4 +256,8 @@ private Path createHome() throws IOException {
         return home;
     }
 
+    private static ESTestCase.TestAnalysis createTestAnalysisFromSettings(Settings settings) throws IOException {
+        return AnalysisTestsHelper.createTestAnalysisFromSettings(settings, new CommonAnalysisPlugin());
```

Comment:
yes. it is a tiny helper method to reduce the line length where it is being used. (because also including `new CommonAnalysisPlugin())` would in many cases cross the 140 character limit)